packages/nextra-theme-docs/src/components/callout.tsx

```diff
@@ -39,9 +39,7 @@ export function Callout({
       >
         {emoji}
       </div>
-      <span className="w-[calc(100%-2.5rem)]">
-        {children}
-      </span>
+      <div className="min-w-0">{children}</div>
```

Comment:
improvement of https://github.com/shuding/nextra/pull/746 to avoid dependence on padding size (`2.5rem`)